src/main/java/edu/kis/vh/nursery/IntArrayStack.java

```diff
@@ -0,0 +1,43 @@
+package edu.kis.vh.nursery;
+
+public class IntArrayStack {
+	
+	private static final int empty_stack = -1;
+	private static final int full_stack = 11;
```

Comment:
This constant is related to stack_capacity. Changing only one leads to an error (inconsistency).